Protocol.md

```diff
@@ -221,11 +255,13 @@ Frame Contents
      * (__M__)etadata: Metadata present
      * (__L__)ease: Will honor LEASE (or not).
      * (__S__)trict: Adhere to strict interpretation of Data and Metadata.
+     * (__R__)esume Enable: Client requests resume capability is possible (Optional).
 * __Major Version__: (16) Major version number of the protocol.
 * __Minor Version__: (16) Minor version number of the protocol.
 * __Time Between KEEPALIVE Frames__: Time (in milliseconds) between KEEPALIVE frames that the client will send.
 * __Max Lifetime__: Time (in milliseconds) that a client will allow a server to not respond to a KEEPALIVE before
 it is assumed to be dead.
+* __Resume Identification Token__: (128) Token used for client resume identification. (Optional - set to all 0s if not supported)
```

Comment:
On the question about whether 128 bits is sufficient, do we want to land this PR and then discuss that separately? For example, should this support variable length?

Or are we comfortable at this point with the use of 128 bits, and the fact that we can always change it in future versions?